fastlane_bot/tools/invariants/vector.py

```diff
@@ -101,6 +101,22 @@ def _kwargs(self, other=None):
         """
         return dict()   
     
+    @property
+    def elements(self):
+        """returns the elements (keys!) of the vector as a list"""
+        return list(self.vec.keys())
+    el = elements
```

Comment:
This global scope line looks out of place (did you really intend for it to be here?).

Reply:
yeah; `el` is simply an alias for `elements`, typically for interactive use; I do that quite often